pkg/pillar/cmd/upgradeconverter/upgradeconverterutils.go

```diff
@@ -0,0 +1,35 @@
+package upgradeconverter
+
+import (
+	log "github.com/sirupsen/logrus"
+	"os"
+	"time"
+)
+
+var OldVersion = "/persist/config/OldGlobalConfig/global.json"
+var NewVersion = "/persist/config/GlobalConfigV2/global.json"
```

Comment:
Take this information in a Json file as input.. Essentially, add a second input:
    upgradeconverter --datafile=<some datafile> --action=convert

This way, it would be very easy to unit test this. Can you can call run directly in the unit test with your data - isn;t it?